src/views/product-root-docs-path-landing/utils/prepare-marketing-blocks.ts

```diff
@@ -0,0 +1,16 @@
+import { serialize } from 'next-mdx-remote/serialize'
+
+/**
+ * Prepares marketing content blocks for rendering on the client.
+ */
+export async function prepareMarketingBlocks(blocks) {
```

Comment:
Right now the only thing we do here is serialize MDX content. I could see potential additions here though, for example if we want to support tutorial cards on docs landing views in the future.

Random note: I think we could potentially group `marketingContentBlocksWithHeadingLevels` into this file as well, to trim down the complexity in the main `server` file, and keep the "marketing block" stuff all together. Again, felt a bit outside the scope of this PR.